src/main/java/com/stripe/model/BalanceTransactionSource.java

```diff
@@ -1,3 +1,3 @@
 package com.stripe.model;
 
-public interface BalanceTransactionSource extends StripeObjectInterface, HasId {}
+public interface BalanceTransactionSource extends HasId {}
```

Comment:
are all those changes normal?

Reply:
No, you'll need to regenerate with the latest codgen.